pkg/apis/admissionregistration/types.go

```diff
@@ -97,7 +98,10 @@ type Rule struct {
 	// '*/scale' means all scale subresources.
 	// '*/*' means all resources and their subresources.
 	//
-	// If '*' or '*/*' is present, the length of the slice must be one.
+	// If wildcard is present, the validation rule will ensure resources do not
+	// overlap with each other.
+	//
+	// Depending on the enclosing object, subresources might not be allowed.
```

Comment:
Doesn't have to be in this PR, but we this is a great example use case for just making two separate InitializerRule and AdmissionWebhookRule structs.